.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest]
+        perl: ['5']
+        include:
+          - { os: 'ubuntu-latest', perl: "5.10" }
+          - { os: 'ubuntu-latest', perl: "5.12" }
+          - { os: 'ubuntu-latest', perl: "5.14" }
+          - { os: 'ubuntu-latest', perl: "5.16" }
+          - { os: 'ubuntu-latest', perl: "5.18" }
+          - { os: 'ubuntu-latest', perl: "5.20" }
+          - { os: 'ubuntu-latest', perl: "5.22" }
+          - { os: 'ubuntu-latest', perl: "5.24" }
+          - { os: 'ubuntu-latest', perl: "5.24" , math-bigint: '1.999813' }
+          - { os: 'ubuntu-latest', perl: "5.26" }
+          - { os: 'ubuntu-latest', perl: "5.28" }
+          - { os: 'ubuntu-latest', perl: "5.30" }
+          - { os: 'ubuntu-latest', perl: "5.32" }
+          - { os: 'ubuntu-latest', perl: "5.34" }
+          - { os: 'ubuntu-latest', perl: "5.34" , perl-threaded: true }
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
`checkout` is now at `v3`, so might be helpful to bump this number.

Reply:
Done!